algorithm.go

```diff
@@ -2,6 +2,8 @@ package cose
 
 import (
 	"crypto"
+	_ "crypto/sha256"
+	_ "crypto/sha512"
```

Comment:
Curious why we don't need `crypto/sha384`. It must be imported somewhere.

Reply:
There is no such a package. `sha384` is imported as part of `crypto/sha512`.